ReleaseTooling/Sources/ZipBuilder/ZipBuilder.swift

```diff
@@ -231,30 +230,29 @@ struct ZipBuilder {
         } else if podInfo.isSourcePod {
           let builder = FrameworkBuilder(projectDir: projectDir,
                                          platform: platform,
-                                         includeCarthage: includeCarthage,
                                          dynamicFrameworks: dynamicFrameworks)
-          let (frameworks, carthageFramework, resourceContents) =
+          let (frameworks, resourceContents) =
             builder.compileFrameworkAndResources(withName: podName,
                                                  logsOutputDir: paths.logsOutputDir,
+                                                 setCarthage: false,
                                                  podInfo: podInfo)
           groupedFrameworks[podName] = (groupedFrameworks[podName] ?? []) + frameworks
-          if platform == .iOS {
-            if let carthageFramework = carthageFramework {
-              carthageToInstall[podName] = [carthageFramework]
-            }
+
+          if podName == "FirebaseCoreDiagnostics" {
```

Comment:
Do we want to also gate this on `includeCarthage`? Otherwise we'll be compiling CD when we don't need to. Same in the below few blocks

Reply:
Any reason, we shouldn't always build Carthage when we're building a Firebase distro. How about removing the `includeCarthage` flag in a subsequent PR?

Reply:
SGTM to move in a subsequent PR and merging as is.